rego/Drive.rego

```diff
@@ -184,28 +303,47 @@ if {
 tests contains {
     "PolicyId": "GWS.DRIVEDOCS.1.4v0.1",
     "Criticality": "Shall",
-    "ReportDetails": utils.ReportDetailsOUs(NonCompliantOUs1_4),
-    "ActualValue": {"NonCompliantOUs": NonCompliantOUs1_4},
+    "ReportDetails": utils.ReportDetails(NonCompliantOUs1_4, NonCompliantGroups1_4),
+    "ActualValue": {"NonCompliantOUs": NonCompliantOUs1_4,
+        "NonCompliantGroups": NonCompliantGroups1_4},
     "RequirementMet": Status,
     "NoSuchEvent": false
 }
 if {
     not NoSuchEvent1_4(utils.TopLevelOU)
-    Status := count(NonCompliantOUs1_4) == 0
+    Conditions := {count(NonCompliantOUs1_4) == 0, count(NonCompliantGroups1_4) == 0 }
+    Status := (false in Conditions) == false
 }
+
 #--
 
 #
 # Baseline GWS.DRIVEDOCS.1.5v0.1
 #--
-NonCompliantOUs1_5 contains OU if {
+
+NonCompliantOUs1_5 contains {
+    "Name": OU,
+    "Value": "Published web content is visible to anyone with a link. "
```

Comment:
This isn't exactly what this setting does. Shared content isn't automatically available to everyone with a link, but enabling this setting allows users to make it so if they want.
![image](https://github.com/cisagov/ScubaGoggles/assets/106177711/7d4cdd01-d12c-4cbb-a853-89955abdda1b)

```suggestion
    "Value": "Published web content can be made visible to anyone with a link. "
```